### ISSUE TITLE
tweetnacl has no package conditional map

### ISSUE THREAD

**Issue by bmayen** [jspm/jspm-cli#1811]
Using v0.17.0-beta.14. When I run `jspm build foo.js --node --format cjs node-production-build.js` I get the following error:

```
Error: Package conditional npm:tweetnacl@0.13.3/#:buffer/global has no package conditional map present.
    at node_modules/systemjs-builder/lib/trace.js:232:17
```

This happens in a gulp task which requires `jspm`. Commenting out the require gets past this error.

**Comment by guybedford** [comment]
Fixed in https://github.com/systemjs/builder/commit/009ecf93da39dc1bad0059c3f81e7657bdb852e5 for the next release.

**Comment by guybedford** [comment]
Released in jspm 0.17.0-beta.15.
